diff --git a/glide2gl/src/Glide64/rdp.h b/glide2gl/src/Glide64/rdp.h
index 7d7b88c..5b74fda 100644
--- a/glide2gl/src/Glide64/rdp.h
+++ b/glide2gl/src/Glide64/rdp.h
@@ -48,7 +48,7 @@
 #include "Gfx_1.3.h"
 #include "../Glitch64/glide.h"
 
-void (*_gSPVertex)(uint32_t addr, uint32_t n, uint32_t v0);
+extern void (*_gSPVertex)(uint32_t addr, uint32_t n, uint32_t v0);
 
 extern uint32_t frame_count; // frame counter
 extern uint32_t gfx_plugin_accuracy;
