diff --git a/sdl/Makefile.libretro b/sdl/Makefile.libretro
--- a/sdl/Makefile.libretro
+++ b/sdl/Makefile.libretro
@@ -777,18 +777,18 @@
 	-I$(NP2_PATH)/i386c/ia32/instructions/fpu
 
 SOURCES_C += $(NP2_PATH)/sdl/libretro/libretro.c \
-	$(wildcard $(NP2_PATH)/i386c/*.c) \
-	$(wildcard $(NP2_PATH)/i386c/ia32/*.c) \
-	$(wildcard $(NP2_PATH)/i386c/ia32/instructions/*.c) \
+	$(sort $(wildcard $(NP2_PATH)/i386c/*.c)) \
+	$(sort $(wildcard $(NP2_PATH)/i386c/ia32/*.c)) \
+	$(sort $(wildcard $(NP2_PATH)/i386c/ia32/instructions/*.c)) \
 	$(NP2_PATH)/i386c/ia32/instructions/fpu/fpdummy.c \
 	$(NP2_PATH)/i386c/ia32/instructions/fpu/fpemul_dosbox.c \
 	$(NP2_PATH)/i386c/ia32/instructions/fpu/fpemul_dosbox2.c \
 	$(NP2_PATH)/i386c/ia32/instructions/fpu/fpemul_softfloat.c \
-	$(wildcard $(NP2_PATH)/i386c/ia32/instructions/fpu/softfloat/*.c) \
-	$(wildcard $(NP2_PATH)/i386c/ia32/instructions/mmx/*.c) \
-	$(wildcard $(NP2_PATH)/i386c/ia32/instructions/sse/*.c) \
-	$(wildcard $(NP2_PATH)/i386c/ia32/instructions/sse2/*.c) \
-	$(wildcard $(NP2_PATH)/i386c/ia32/instructions/sse3/*.c)
+	$(sort $(wildcard $(NP2_PATH)/i386c/ia32/instructions/fpu/softfloat/*.c)) \
+	$(sort $(wildcard $(NP2_PATH)/i386c/ia32/instructions/mmx/*.c)) \
+	$(sort $(wildcard $(NP2_PATH)/i386c/ia32/instructions/sse/*.c)) \
+	$(sort $(wildcard $(NP2_PATH)/i386c/ia32/instructions/sse2/*.c)) \
+	$(sort $(wildcard $(NP2_PATH)/i386c/ia32/instructions/sse3/*.c))
 
 #	$(NP2_PATH)/sdl/libretro/libretro-common/file/archive_file_zlib.c \
 #	$(NP2_PATH)/sdl/libretro/libretro-common/formats/libchdr/libchdr_flac.c \
