diff --git a/svn-current/trunk/makefile.libretro b/svn-current/trunk/makefile.libretro
--- a/svn-current/trunk/makefile.libretro
+++ b/svn-current/trunk/makefile.libretro
@@ -652,11 +652,11 @@
    BURN_BLACKLIST += $(FBA_BURNER_DIR)/un7z.cpp
 endif
 
-SOURCES_CXX += $(GRIFFIN_CXXSRCFILES) $(filter-out $(BURN_BLACKLIST),$(foreach dir,$(FBA_SRC_DIRS),$(wildcard $(dir)/*.cpp)))
+SOURCES_CXX += $(GRIFFIN_CXXSRCFILES) $(filter-out $(BURN_BLACKLIST),$(foreach dir,$(FBA_SRC_DIRS),$(sort $(wildcard $(dir)/*.cpp))))
 SOURCES_CXX += $(LIBRETRO_DIR)/libretro.cpp \
 	$(LIBRETRO_DIR)/retro_common.cpp \
 	$(LIBRETRO_DIR)/retro_input.cpp
-SOURCES_C += $(filter-out $(BURN_BLACKLIST),$(foreach dir,$(FBA_SRC_DIRS),$(wildcard $(dir)/*.c)))
+SOURCES_C += $(filter-out $(BURN_BLACKLIST),$(foreach dir,$(FBA_SRC_DIRS),$(sort $(wildcard $(dir)/*.c))))
 
 FBA_CXXOBJ := $(SOURCES_CXX:.cpp=.o)
 FBA_COBJ := $(SOURCES_C:.c=.o)
